site/_data/shortlinks.yml

```diff
@@ -22,6 +22,3 @@
 - title: Make your own plugins
   key: examples
   destination: examples
-- title: Cut a release
```

Comment:
I'm removing this but I don't think it has an effect since the side bar has already been reorganised.

Reply:
docs/master/release still works. I'm not really sure what this file is for.